Subject: Remove redundant, obsolete 2.4 compatibility code in wlan-ng
From: Sven-Thorsten Dietrich <sdietrich@suse.de>

Fix genirq compile error in drivers/staging/wlan-ng/prism2_usb
introduced by:
patches.drivers/0015-Staging-add-wlan-ng-prism2-usb-driver.patch

Acked-by: Sven-Thorsten Dietrich <sdietrich@suse.de>
---
 drivers/staging/wlan-ng/wlan_compat.h |    8 --------
 1 file changed, 8 deletions(-)

Index: linux-2.6.27-genirq/drivers/staging/wlan-ng/wlan_compat.h
===================================================================
--- linux-2.6.27-genirq.orig/drivers/staging/wlan-ng/wlan_compat.h
+++ linux-2.6.27-genirq/drivers/staging/wlan-ng/wlan_compat.h
@@ -589,14 +589,6 @@ struct iw_request_info
 #define _MODULE_PARM_STRING_bool "i"
 #endif
 
-/* linux < 2.5.69 */
-#ifndef IRQ_NONE
-typedef void irqreturn_t;
-#define IRQ_NONE
-#define IRQ_HANDLED
-#define IRQ_RETVAL(x)
-#endif
-
 #ifndef in_atomic
 #define in_atomic()  0
 #endif
